docs/index.rst

```diff
@@ -1,5 +1,4 @@
-:mod:`zope.container`
-=====================
+.. include:: ../README.rst
```

Comment:
I'm a little anti-DRY about this.  The text of the README may be (or become) unsuitable for this use: It tends to become a bit "developert-centric" in most of our repos (e.g., badges, which may then include links back to RTD, etc.)